spec/requests/quizzes_spec.rb

```diff
@@ -63,6 +76,18 @@
     end
   end
 
+  describe "PATCH update with invalid parameters" do
+    let(:quiz) { Quiz.create!(title: "Valid Quiz", description: "Valid Description") }
+
+    it "does not update the quiz and displays error messages" do
+      patch quiz_path(quiz.id), params: { quiz: { title: "", description: "" } }
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include("There were errors with your submission. Please fix them below.")
+      expect(response.body).to include("Title can&#39;t be blank")
+      expect(response.body).to include("Description can&#39;t be blank")
```

Comment:
These are peculiar - they don't match the validation messages you've configured in your model - are we sure this is being run?